app_sensor_module/src/main.c

```diff
@@ -16,6 +16,7 @@
 #include "sensors.h"
 
 LOG_MODULE_REGISTER(main, CONFIG_APP_LOG_LEVEL);
+LOG_MODULE_REGISTER(tmp116_task, LOG_LEVEL_INF);
```

Comment:
Remove this. Should not have these in multiple files technically and not have this show multiple times in a single file